region/maridia/outer/Main Street.json

```diff
@@ -2899,6 +2899,33 @@
         "Near the end, use the global crab to get onto the top left corner of the room, then jump towards the walkway leading to the door."
       ],
       "devNote": "The global crab will fall if the speed blocks are broken."
+    },
+    {
+      "link": [9, 4],
+      "name": "Scisor Double Damage Boost",
+      "requires": [
+        "h_canNavigateUnderwater",
+        {"obstaclesNotCleared": ["A"]},
+        "canCrouchJump",
+        "canInsaneJump",
+        "canReserveDoubleDamageBoost",
+        {"or": [
+          "canTrickySpringBallJump",
+          "HiJump"
+        ]},
+        {"autoReserveTrigger": {"minReserveEnergy": 85}},
```

Comment:
Assuming this 85 value is based on how long the i-frames last; is it basically the same for all enemies?

Reply:
Yes, it is based on how long it takes for Samus' IFrames to run out.  The `85` was taken from the existing double reserve boost strat, I'm not sure what you actually need.  
You'll additionally (simultaneously?) need enough reserve energy to survive the crab hit and enough Etank capacity to hold that energy.

Reply:
I finally got it. With my savestate, I changed my reserves and found that it worked with as low as 74 reserve energy. I also found that spamming pause to fix the black camera asap was very helpful in preventing me from taking one extra hit after being on the ledge.

Reply:
Does this need Morph? I only tested it with.

Reply:
Its going to be much easier with Morph... in pre cathedral you need to do a blind damage boost to gain the height without morph, but this is underwater too.  Tas could probably do it 🤷 

And yeah, you want to repause asap so you can see again.

My frame advancing showed 95 frames for reserves to completely eliminate iframes.  There are some iframes used as part of the first boost while you are dead and pausing.  And there are some spare frames from the height gain not needing to be completely perfect.  And if you do not have enough reserve energy (regarding iframes), the pause can be moved earlier so that iframes are active while you gain the remainder of the first damage boost and position for the second, theoretically.
Its going to be better to not be too exact with the required reserve energy; and more commonly you would want all reserves ~full.